<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>message_widget</class>
 <widget class="QWidget" name="message_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame:hover {
	background-color: #303030;
}

#btn_avatar {
	background: none;
	border: none;
}

#label_username {
	font-size: 11pt;
	font-weight: 600;
	color: black;
}

#label_timestamp {
	margin-left: 3px;
	margin-bottom: -3px;
	font-size: 9pt;
	color: #808080;
}</string>
  </property>
  <layout class="QVBoxLayout" name="layout_widget">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::TabFocus</enum>
     </property>
     <layout class="QGridLayout" name="layout_frame" rowstretch="0,0,1" columnstretch="0,0,1" columnminimumwidth="48,0,0">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string notr="true">Username</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_timestamp">
        <property name="text">
         <string notr="true">0:00</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QFrame" name="frame_contents">
        <layout class="QGridLayout" name="layout_contents">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>3</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="FrameButton" name="btn_avatar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="layout_avatar_btn">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MaskedImage" name="icon_avatar">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../resources/index.qrc">:/icons/avatar</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QVBoxLayout" name="layout_reply">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameButton</class>
   <extends>QFrame</extends>
   <header>widgets.common</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MaskedImage</class>
   <extends>QLabel</extends>
   <header>widgets.common</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/index.qrc"/>
  <include location="../../../resources/index.qrc"/>
  <include location="../../../resources/index.qrc"/>
 </resources>
 <connections/>
</ui>
